packages/autocannon-ui-frontend/src/components/ResultSet.jsx

```diff
@@ -47,6 +52,7 @@ export default function ResultSet({ data }) {
       <Accordion defaultExpanded>
         <AccordionSummary
           expandIcon={<ExpandMoreIcon />}
+          className={classes.summary}
```

Comment:
comment is about line 60. xs is not a valid property on a Grid container element. Please remove, you'll see this causes a JS error when running the app and executing a run